moveit_ros/planning/planning_scene_monitor/src/current_state_monitor.cpp

```diff
@@ -163,36 +168,41 @@ void planning_scene_monitor::CurrentStateMonitor::stopStateMonitor()
 {
   if (state_monitor_started_)
   {
-    joint_state_subscriber_.shutdown();
+    rclcpp::shutdown();
```

Comment:
this is the wrong kind of shutdown. we don't want to turn off the entire node!

Reply:
Yes, I already had some changes for that, using ``joint_state_subscriber_.reset()``